### PR TITLE
Update references (#457).

### LINKED ISSUE #457: Review references.
See also #293

NORMATIVE REFERENCES:

[GPS] This cite is to an unusual source. But for both UTC and GPS, a reference to the technology is not really all that helpful. It’s only important that one know unambiguously which it is.  The one required bit of information needed by transformation processors is the current in force leap seconds, which is not listed. It turns out that an authoritative normative reference for this has an equally sketchy pedigree. I think GPS and UTC references are arguably both not essential except for the leap second aspect.

[HTML5] - As we all know, the real spec for HTML5 is at whatwg.org.  Although citing the old W3C Rec is “tidy”, it’s not technically accurate. At least add an informative reference to whatwg and consider what to do longer term.

[SMPTE 12M] – Obsoleted by: ST 12-1:2008 - SMPTE Standard - For Television — Time and Control Code, http://ieeexplore.ieee.org/document/7289820/ 

[URI] – Obsoleted by RFC 3986.  Also, in general, the more helpful RFC URLs are now here, e.g. https://tools.ietf.org/html/rfc3986 

OTHER REFERENCES:

[CMAF] is not yet published and at best is the first of the year.  But the citation in 12.4.1 is a bit misleading.  CMAF just points to 14496-30 and defines some signaling information. I would either remove this or add additional context about why it is relevant.

[ISOBMFF] – Although just a label, the rest of the world uses this acronym to mean specifically, 14496-12.  I’d recommend changing it to “ISO-TT” or something else to avoid reader confusion.

[SMPTE 170M] – Obsoleted by: ST 170:2004 - SMPTE Standard - For Television — Composite Analog Video Signal — NTSC for Studio Applications, http://ieeexplore.ieee.org/document/7291416/ 

[SMPTE 2052-11] – use the full title: RP 2052-11:2013 - SMPTE Recommended Practice - Conversion from CEA-708 Caption Data to SMPTE-TT and add: http://ieeexplore.ieee.org/document/7290363/ 

GENERAL:

I think dates/versions are generally essential on normative references with the exception of reference to code point registries. 

Conversely, using undated informative references is OK as long as the title is not likely to change. But to that point, note the SMTPE title changes. Readers can always decide to read newer versions of informative references.

Wherever possible it is good to provide a URL to acquire the citations.

### PR DESCRIPTION
**Pull request by skynavga**
Closes #457.

### CODE REVIEW

**Comment by cconcolato** [comment]
LGTM

File: spec/ttml2.xml [hunk 1]
```diff
@@ -24926,9 +24925,8 @@ Multimedia Integration Language (SMIL 3.0)</titleref>,
 W3C Recommendation, 1 December 2008. (See
 <xspecref href="http://www.w3.org/TR/2008/REC-SMIL3-20081201/">http://www.w3.org/TR/2008/REC-SMIL3-20081201/</xspecref>.)
 </bibl>
-<bibl id="smpte12m" key="SMPTE 12M">ANSI/SMPTE 12M:1999,
-<titleref href="https://www.smpte.org/">Television, Audio and Film &ndash;
-Time and Control Code</titleref>, SMPTE Standard.
+<bibl id="smpte12" key="SMPTE ST12-1">SMPTE ST12-1:2014,
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: In #457 @mikedo suggests using the 2008 version. @mikedo is there a reason you did not choose the 2014 version?

**Comment by skynavga** [comment]
On Tue, Jan 2, 2018 at 4:26 PM, Nigel Megitt <notifications@github.com>
wrote:

> *@nigelmegitt* requested changes on this pull request.
>
> There's a discrepancy here with the issue, but I do not know if it has any
> impact - the issue mentions SMPTE 12-1:2008 but the pull request uses the
> 2014 version.
>
> There are also some references *not* updated as per the issue, i.e. SMPTE
> 170M and 2052-11, so as far as I can tell this pull request does not
> resolve the issue.
> ------------------------------
>
> In spec/ttml2.xml
> <https://github.com/w3c/ttml2/pull/522#discussion_r159337775>:
>
> > @@ -24926,9 +24925,8 @@ Multimedia Integration Language (SMIL 3.0)</titleref>,
>  W3C Recommendation, 1 December 2008. (See
>  <xspecref href="http://www.w3.org/TR/2008/REC-SMIL3-20081201/">http://www.w3.org/TR/2008/REC-SMIL3-20081201/</xspecref>.)
>  </bibl>
> -<bibl id="smpte12m" key="SMPTE 12M">ANSI/SMPTE 12M:1999,
> -<titleref href="https://www.smpte.org/">Television, Audio and Film &ndash;
> -Time and Control Code</titleref>, SMPTE Standard.
> +<bibl id="smpte12" key="SMPTE ST12-1">SMPTE ST12-1:2014,
>
> In #457 <https://github.com/w3c/ttml2/issues/457> @mikedo
> <https://github.com/mikedo> suggests using the 2008 version. @mikedo
> <https://github.com/mikedo> is there a reason you did not choose the 2014
> version?
>
The changes for 170 and 2052-11 were inadvertently left out. I will add. I
update the date to the 2014 edition as the 2008 edition is shown as
superseded.

**Comment by skynavga** [comment]
@nigelmegitt fixed 170, 2052-11

**Pull request merged**